### DISCUSSION TITLE
Adding a Separator

### DISCUSSION THREAD

**Discussion by Nadias-jp** [mpdifran/Memento-Issues#39] [Ideas]
If I can insert a separator anywhere, it will be possible to use it like Kanban View.

It is useful enough to be able to sort, but I think it will be much easier to see if I can use separators.

LIST1
 Task1 Task2 Task3 <-Priority
 ------------------ <-Separator
 Task 4 Task 5

If I try to do this with lists, the number of lists will double.

**Comment by mpdifran** [comment]
Thanks for the suggestion! I try to keep Memento simple, but I'll see if I can make this work.
